src/Tools/dotnet-monitor/CollectionRules/Actions/CollectTraceAction.cs

```diff
@@ -90,7 +94,19 @@ public CollectTraceAction(IServiceProvider serviceProvider, IEndpointInfo endpoi
                     async (outputStream, token) =>
                     {
                         using IDisposable operationRegistration = _operationTrackerService.Register(EndpointInfo);
-                        await TraceUtilities.CaptureTraceAsync(startCompletionSource, EndpointInfo, configuration, duration, outputStream, token);
+                        if (null != stoppingEvent)
+                        {
+                            ILogger<CollectTraceAction> logger = _serviceProvider
+                                .GetRequiredService<ILoggerFactory>()
+                                .CreateLogger<CollectTraceAction>();
+                            using var _ = logger.BeginScope(scope);
```

Comment:
I think this is adding a duplicated scope since the egress operation should be logging this already.